pkg/io/fileWriter_test.go

```diff
@@ -0,0 +1,40 @@
+package io
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMakeDirForFile_HappyPath(t *testing.T) {
+	file, err := ioutil.TempFile("", "test")
+	require.NoError(t, err)
+	filePath := file.Name()
+
+	dir := path.Dir(filePath)
```

Comment:
Just use `TempDir` if you want to get a temp dir, otherwise you may get `/tmp` here which is wrong.